libnopegl/src/backends/gl/gpu_ctx_gl.c

```diff
@@ -1039,11 +1039,11 @@ static void gl_set_pipeline(struct gpu_ctx *s, struct gpu_pipeline *pipeline)
 {
 }
 
-static void gl_draw(struct gpu_ctx *s, int nb_vertices, int nb_instances)
+static void gl_draw(struct gpu_ctx *s, int nb_vertices, int nb_instances, int first_vertex)
```

Comment:
I feel like the argument should be before `nb_vertices` as to depict a start/count range. GL and wgpu do that (ok, not Vulkan). It feels weird to have `vertex-param, instance-param, vertex-param`

Reply:
I'm not opposed to the change. However, can I do it when all the GPU code refactoring is done ? (and avoid too much conflicts).